styles/components/news.less

```diff
@@ -64,7 +64,7 @@
 }
 
 .news-content p {
-    text-indent: 40px;
+    margin-left: 40px;
```

Comment:
Mhh... This was originally meant as a paragraph indentation https://en.wikipedia.org/wiki/Indentation_(typesetting)

But it seems to annoy you so let's just get rid of the margin entirely.

Reply:
The intendation is great when used on multi line text. But our page is one <p> after another and tus also single lines getting 40px indented. Which results in only on line of "real" left aligned floating text. looking very awkward. Like lists that are broken and do not indent the next line.
I am going to redesign to multiple columns. and maybe paragraph indents will make sense when the floating text is better consolidated into fewer paragraphs. Lets see.